CRAFT-2027-Export messages to `NimbusProvider` (#837)
commercetools/nimbus: packages/nimbus/src/components/nimbus-provider/nimbus-provider.mdx

}
```

## Set up translation messages

> To enable translations for Nimbus components, you need to load and provide translation messages to the `NimbusProvider`. This is a **one-time setup at your application's root level**. Once configured, all Nimbus components throughout your application will automatically use the correct translations. Nimbus provides utilities to load these messages from the `@commercetools/nimbus-i18n` package.

Reviewer: ```suggestion
To enable internationalized messages in Nimbus components, you need to load and provide translation messages to the `NimbusProvider`. This is a **one-time setup at your application's root level**. Once configured, all Nimbus components throughout your application will automatically render the correct internationalized messages. Nimbus provides utilities to load these messages from the `@commercetools/nimbus-i18n` package.
```
